### PR TITLE
chore: new default set of events

### PR DESCRIPTION
**Pull request by geyslan**
7f860ae2 chore: new default set of events

Fixes:

- #2631 

## Type of change

- [x] Breaking change (cause existing functionality not to work as expected).

### CODE REVIEW

**Comment by rafaeldtinoco** [comment]
@yanivagman this one is yours, I believe it needs a decision more than a technical view.

File: pkg/events/events.go [hunk 1]
```diff
@@ -1387,7 +1387,7 @@ var Definitions = eventDefinitions{
 			ID32Bit: sys32setuid32,
 			Name:    "setuid",
 			Syscall: true,
-			Sets:    []string{"default", "syscalls", "proc", "proc_ids"},
+			Sets:    []string{"syscalls", "proc", "proc_ids"},
```
Comment (yanivagman) [inline pkg/events/events.go:1390]: Are the other setXXXid syscalls noisy? Maybe we should include them after all?

File: pkg/events/events.go [hunk 2]
```diff
@@ -1960,7 +1960,7 @@ var Definitions = eventDefinitions{
 			ID32Bit: sys32mount,
 			Name:    "mount",
 			Syscall: true,
-			Sets:    []string{"default", "syscalls", "fs"},
+			Sets:    []string{"syscalls", "fs"},
```
Comment (yanivagman) [inline pkg/events/events.go:1963]: I think we should keep this one. It is important enough and not too frequent

File: pkg/events/events.go [hunk 3]
```diff
@@ -2061,7 +2061,7 @@ var Definitions = eventDefinitions{
 			ID32Bit: sys32init_module,
 			Name:    "init_module",
 			Syscall: true,
-			Sets:    []string{"default", "syscalls", "system", "system_module"},
+			Sets:    []string{"syscalls", "system", "system_module"},
```
Comment (yanivagman) [inline pkg/events/events.go:2064]: init_module is important enought to be included

File: pkg/events/events.go [hunk 4]
```diff
@@ -2950,7 +2950,7 @@ var Definitions = eventDefinitions{
 			ID32Bit: sys32fchownat,
 			Name:    "fchownat",
 			Syscall: true,
-			Sets:    []string{"default", "syscalls", "fs", "fs_file_attr"},
+			Sets:    []string{"syscalls", "fs", "fs_file_attr"},
```
Comment (yanivagman) [inline pkg/events/events.go:2953]: fchownat should be kept if we are adding chownat
if we keep chown then fchownat should be kept as well

File: pkg/events/events.go [hunk 5]
```diff
@@ -3529,7 +3529,7 @@ var Definitions = eventDefinitions{
 			ID32Bit: sys32process_vm_readv,
 			Name:    "process_vm_readv",
 			Syscall: true,
-			Sets:    []string{"default", "syscalls", "proc"},
+			Sets:    []string{"syscalls", "proc"},
```
Comment (yanivagman) [inline pkg/events/events.go:3532]: let's keept this one too

File: pkg/events/events.go [hunk 6]
```diff
@@ -3570,7 +3570,7 @@ var Definitions = eventDefinitions{
 			ID32Bit: sys32finit_module,
 			Name:    "finit_module",
 			Syscall: true,
-			Sets:    []string{"default", "syscalls", "system", "system_module"},
+			Sets:    []string{"syscalls", "system", "system_module"},
```
Comment (yanivagman) [inline pkg/events/events.go:3573]: and this

File: pkg/events/events.go [hunk 7]
```diff
@@ -1190,7 +1190,7 @@ var Definitions = eventDefinitions{
 			ID32Bit: sys32creat,
 			Name:    "creat",
 			Syscall: true,
-			Sets:    []string{"default", "syscalls", "fs", "fs_file_ops"},
+			Sets:    []string{"syscalls", "fs", "fs_file_ops"},
```
Comment (yanivagman) [inline pkg/events/events.go:1193]: let's keep this to track the creation of new files

File: pkg/events/events.go [hunk 8]
```diff
@@ -1282,7 +1282,7 @@ var Definitions = eventDefinitions{
 			ID32Bit: sys32lchown32,
 			Name:    "lchown",
 			Syscall: true,
-			Sets:    []string{"default", "syscalls", "fs", "fs_file_attr"},
+			Sets:    []string{"syscalls", "fs", "fs_file_attr"},
```
Comment (yanivagman) [inline pkg/events/events.go:1285]: if we keep chown then we should keep this as well

File: pkg/events/events.go [hunk 9]
```diff
@@ -3639,7 +3639,7 @@ var Definitions = eventDefinitions{
 			ID32Bit: sys32memfd_create,
 			Name:    "memfd_create",
 			Syscall: true,
-			Sets:    []string{"default", "syscalls", "fs", "fs_file_ops"},
+			Sets:    []string{"syscalls", "fs", "fs_file_ops"},
```
Comment (yanivagman) [inline pkg/events/events.go:3642]: we should also keep this one

File: pkg/events/events.go [hunk 10]
```diff
@@ -3891,7 +3891,7 @@ var Definitions = eventDefinitions{
 			ID32Bit: sys32move_mount,
 			Name:    "move_mount",
 			Syscall: true,
-			Sets:    []string{"default", "syscalls", "fs"},
+			Sets:    []string{"syscalls", "fs"},
```
Comment (yanivagman) [inline pkg/events/events.go:3894]: let's keep it

File: pkg/events/events.go [hunk 11]
```diff
@@ -5398,7 +5398,7 @@ var Definitions = eventDefinitions{
 			Probes: []probeDependency{
 				{Handle: probes.SecuritySbMount, Required: true},
 			},
-			Sets: []string{"default", "lsm_hooks", "fs"},
+			Sets: []string{"lsm_hooks", "fs"},
```
Comment (yanivagman) [inline pkg/events/events.go:5401]: let's keep it

**Review by yanivagman — CHANGES_REQUESTED** [review]
Some events we should reconsider adding. WDYT?

File: pkg/events/events.go (hunk 1 above)
Comment (geyslan) [reply pkg/events/events.go:1390]: `sudo nano /etc/shadow` generated 22 events with all setXXXid, so it seems ok. Including them.

**Review by yanivagman — APPROVED** [review]
LGTM

**Pull request merged**